applications_tests/lethe-fluid/mms2d_cahn_hilliard.prm

```diff
@@ -40,7 +40,7 @@ subsection analytical solution
   set verbosity = verbose
 
   subsection cahn hilliard
-    set Function expression = sin(pi*x)*sin(pi*y); 1/0.0 #This is to make the test more robust on every configuration. Only the phase (first component) is being tested, the potential is not tested, the numerical solution converges to zero extremely fast.
+    set Function expression = -sin(pi*x)*sin(pi*y); 1/0.0 #This is to make the test more robust on every configuration. Only the phase (first component) is being tested, the potential is not tested, the numerical solution converges to zero extremely fast.
```

Comment:
Since I am only interested in the phase order parameter, the chemical potential acts rather chaotic. To avoid the test to fail on other configurations, I set the analytical solution for the chemical potential to 1/0.0 so that it always print nan. By robust I meant that every configuration should be able to run this test, and I left the disclaimer because people might be surprised by the 1/0.0 term.

Reply:
OKidoke